<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="stop">
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Monitor ON/OFF</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>940</width>
         <height>521</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>摄像头/视频</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="DeviceConnect">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string>Device Connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menumonitor">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionRead_Camera"/>
    <addaction name="actionOpen_Camera"/>
    <addaction name="actionCalculate_FPS"/>
    <addaction name="actionExit"/>
    <addaction name="actionDevice"/>
   </widget>
   <addaction name="menumonitor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRead_Camera">
   <property name="text">
    <string>Camera Infomation</string>
   </property>
  </action>
  <action name="actionOpen_Camera">
   <property name="text">
    <string>Open Camera</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCalculate_FPS">
   <property name="text">
    <string>Calculate FPS</string>
   </property>
  </action>
  <action name="actionDevice">
   <property name="text">
    <string>Device Connect</string>
   </property>
   <property name="toolTip">
    <string>Device Connect</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
